charts/operator/values.yaml

```diff
@@ -152,6 +159,26 @@ monitoring:
 # Sets the name of the Secret in which to store the self-signed TLS certificate for the Webhooks when `webhook.enabled` is `true`.
 webhookSecretName: webhook-server-cert
 
+# -- Sets almost all fields of operator Deployment PodTemplateSpec
```

Comment:
Can we copy the documentation that's within the redpanda chart? IIRC that should include a few examples as well.

Reply:
Agree